app/component/CustomizeSearch.js

```diff
@@ -29,6 +32,16 @@ function mapToSlider(value, arr) {
   return best;
 }
 
+// Get default settings
+const defaultSettings = {
```

Comment:
In our case the 'defaults' are varying site by site, for example in opas.matka.fi we also have AIRPLANE active by default.

getDefaultModes() returns the currently active set of default modes.